### DISCUSSION TITLE
WGC Yellow Border on Windows 10

### DISCUSSION THREAD

**Discussion by nohbodyyouknow** [WindowTop/WindowTop-App#377] [Q&A] [answered]
I'm having an issue where I am stuck with the yellow windows capture border surrounding a program, when I use Glass mode. It only happens with Glass mode and not Opacity. I am attempting to use this to make my LiveSplit timer overlay have a transparent background but visible text, which it does perfectly, but I'm left with a yellow border surrounding the transparent window. Does anyone have a fix? I updated to the most recent version of Windows 10 as suggested, which is version 22h2 (OS Build 19045.4894). Thanks in advance for your time.
![image](https://github.com/user-attachments/assets/0cc379ee-d670-456c-b3da-d575a5d9b3c2)

**Comment by gileli121** [comment, marked as answer]
In that case, it is a bug.
The most recent Windows 10 version does support disabling the yellow border via API.
WindowTop probably does not use it because it thinks that it does not support it.

I opened a bug 
https://github.com/WindowTop/WindowTop-App/issues/378

**Reply by nohbodyyouknow** [reply]
Thanks for the quick reply and escalating this to a bug report gileli. I appreciate it! ✌
